specification/videoanalyzer/resource-manager/Microsoft.Media/preview/2021-05-01-preview/Videos.json

```diff
@@ -0,0 +1,1035 @@
+          },
+          "204": {
+            "description": "NoContent"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/AzureVideoAnalyzerAccountName"
+          },
+          {
+            "name": "videoName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the video to delete."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Deletes a video entity.": {
+            "$ref": "examples/video-delete.json"
+          }
+        }
+      },
+      "patch": {
+        "summary": "Updates the properties of a video resource.",
+        "description": "Updates individual properties of an existing video resource.",
+        "operationId": "Videos_Update",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VideoEntity"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/AzureVideoAnalyzerAccountName"
+          },
+          {
+            "name": "videoName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the video to update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VideoEntity"
+            },
+            "description": "The request parameters"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Update video entity.": {
+            "$ref": "examples/video-patch.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/videoAnalyzers/{accountName}/videos/{videoName}/listStreamingToken": {
+      "post": {
+        "summary": "Generates a streaming token for video playback.",
```

Comment:
>Generates [](start = 20, length = 9)

nit: Generate? or List

Reply:
even though the API says list, its a post operation, which generates new token every time.